crates/context_aware_config/src/api/config/handlers.rs

```diff
@@ -536,8 +553,97 @@ async fn reduce_config(
+    state: Data<AppState>,
+) -> superposition::Result<HttpResponse> {
+    use fred::interfaces::MetricsInterface;
+
+    log::debug!("Started redis fetch");
+    let config_key = format!("{}::cac_config", *tenant);
+    let last_modified_at_key = format!("{}::cac_config::last_modified_at", *tenant);
+    let audit_id_key = format!("{}::cac_config::audit_id", *tenant);
+    let config_version_key = format!("{}::cac_config::config_version", *tenant);
+    let client = state.redis.next_connected();
+    let config = client.get::<String, String>(config_key).await;
+    let metrics = client.take_latency_metrics();
+    let network_metrics = client.take_network_latency_metrics();
+    log::trace!(
+        "Network metrics for config fetch in milliseconds :: max: {}, min: {}, avg: {}; Latency metrics :: max: {}, min: {}, avg: {}",
+        network_metrics.max,
+        network_metrics.min,
+        network_metrics.avg,
+        metrics.max,
+        metrics.min,
+        metrics.avg
+    );
+    match config {
+        Ok(config) => {
+            let mut response = HttpResponse::Ok();
+            if let Ok(max_created_at) =
+                client.get::<String, String>(last_modified_at_key).await
+            {
+                let metrics = client.take_latency_metrics();
+                let network_metrics = client.take_network_latency_metrics();
+                log::trace!(
+                    "Network metrics max-created-by fetch in milliseconds :: max: {}, min: {}, avg: {}; Latency metrics :: max: {}, min: {}, avg: {}",
+                    network_metrics.max,
+                    network_metrics.min,
+                    network_metrics.avg,
+                    metrics.max,
+                    metrics.min,
+                    metrics.avg
+                );
+                response
+                    .insert_header((AppHeader::LastModified.to_string(), max_created_at));
+            }
+            if let Ok(audit_id) = client.get::<String, String>(audit_id_key).await {
+                let metrics = client.take_latency_metrics();
+                let network_metrics = client.take_network_latency_metrics();
+                log::trace!(
+                    "Network metrics for audit ID in milliseconds :: max: {}, min: {}, avg: {}; Latency metrics :: max: {}, min: {}, avg: {}",
+                    network_metrics.max,
+                    network_metrics.min,
+                    network_metrics.avg,
+                    metrics.max,
+                    metrics.min,
+                    metrics.avg
+                );
+                response.insert_header((AppHeader::XAuditId.to_string(), audit_id));
+            }
+            if let Ok(config_version) =
+                client.get::<Option<i64>, String>(config_version_key).await
+            {
+                let metrics = client.take_latency_metrics();
+                let network_metrics = client.take_network_latency_metrics();
+                log::trace!(
```

Comment:
we have not enabled trace in prod , can we make this info level

Reply:
No required, I want this only on trace debugs